minvws/nl-kat-coordination
# Fix `docs` target in Makefile

### Changes

This PR fixes the `docs` target when used on a Mac machine that doesn't have CoreUtils installed.

### QA notes

- Run `make docs` on a Linux machine and verify there are no build errors
- Run `make docs` on a Mac machine and verify there are no build errors

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
